ci-operator/step-registry/ovn/conf/ipsec-manifest/ovn-conf-ipsec-manifest-commands.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cat >> "${SHARED_DIR}/manifest_cluster-network-03-config.yml" << EOF
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  creationTimestamp: null
+  name: cluster
+spec:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  externalIP:
+    policy: {}
+  networkType: OVNKubernetes
+  serviceNetwork:
+  - 172.30.0.0/16
+  defaultNetwork:
+    type: OVNKubernetes
+    ovnKubernetesConfig:
+      ipsecConfig: {}
+EOF
+
```

Comment:
no problem with me to do one machineconfig for workers and masters, but since they are duplicate
configs, it reminded me of a different way to do it with a bash loop:

https://github.com/openshift/release/pull/41541/commits/c6df29914066414a0f8c0bdc1b30b1bd22be12b5